tests/flamegraph.rs

```diff
@@ -232,6 +232,22 @@ fn flamegraph_nameattr_empty_attribute() {
     test_flamegraph(input_file, expected_result_file, options).unwrap();
 }
 
+#[test]
+fn flamegraph_nameattr_href_no_target() {
```

Comment:
Oh, I was just thinking we could add it to the existing nameattr test. Do you think it's worthwhile adding this as a separate test?